src/content/users/ListView.tsx

```diff
@@ -380,88 +444,22 @@ const ListingView: FC = () => {
-                {columns.map((col: Column) => (
-                  <StyledHeaderCell key={col.label}>
-                    {col.comparator ? (
-                      <TableSortLabel
-                        active={orderBy === col}
-                        direction={orderBy === col ? order : "asc"}
-                        onClick={() => handleRequestSort(col)}
-                      >
-                        {col.label}
-                      </TableSortLabel>
-                    ) : (
-                      col.label
-                    )}
-                  </StyledHeaderCell>
-                ))}
-              </TableRow>
-            </StyledTableHead>
-            <TableBody>
-              {loading && (
-                <TableRow>
-                  <TableCell>
-                    <SuspenseLoader fullscreen={false} />
-                  </TableCell>
-                </TableRow>
-              )}
-              {dataset.map((d: T) => (
-                <TableRow tabIndex={-1} hover key={`${d["_id"]}`}>
-                  {columns.map((col: Column) => (
-                    <StyledTableCell key={`${d["_id"]}_${col.label}`}>
-                      {col.value(d)}
-                    </StyledTableCell>
-                  ))}
-                </TableRow>
-              ))}
-
-              {/* Fill the difference between perPage and count to prevent height changes */}
-              {emptyRows > 0 && (
-                <TableRow style={{ height: 53 * emptyRows }}>
-                  <TableCell colSpan={columns.length} />
-                </TableRow>
-              )}
-
-              {/* No content message */}
-              {(!dataset.length || dataset.length === 0) && (
-                <TableRow style={{ height: 53 * 10 }}>
-                  <TableCell colSpan={columns.length}>
-                    <Typography variant="h6" align="center" fontSize={18} color="#757575">
-                      No users found.
-                    </Typography>
-                  </TableCell>
-                </TableRow>
-              )}
-            </TableBody>
-          </Table>
-          <StyledTablePagination
-            rowsPerPageOptions={[5, 10, 20, 50]}
-            component="div"
-            count={count}
-            rowsPerPage={perPage}
-            page={page}
-            onPageChange={(e, newPage) => setPage(newPage)}
-            onRowsPerPageChange={handleChangeRowsPerPage}
-            nextIconButtonProps={{
-              disabled:
-                perPage === -1 ||
-                !dataset ||
-                dataset.length === 0 ||
-                count <= (page + 1) * perPage ||
-                emptyRows > 0 ||
-                loading,
-            }}
-            SelectProps={{
-              inputProps: { "aria-label": "rows per page" },
-              native: true,
-            }}
-            backIconButtonProps={{ disabled: page === 0 || loading }}
-          />
-        </StyledTableContainer>
+        <GenericTable
```

Comment:
I'm annotating this here because I'm not sure exactly where the issue is – When changing filters or navigating to a different page, the `rowsPerPage` query parameter is added to the URL when it's the default value (20 in this case). 

The impact of this is absolutely nothing, but I just want to mention this because I don't think (?) it's intentional. If it is intentional, you can just dismiss this.

(issue is also on OrganizationView also)